src/main/java/com/example/pomeserver/domain/record/controller/RecordController.java

```diff
@@ -34,6 +36,16 @@ public ApplicationResponse<RecordResponse> findById(@PathVariable Long recordId)
         return recordService.findById(recordId);
     }
 
```

Comment:
```suggestion
/**
 * 회원가입 기능
 * @Author 한규범
 */
```
요런식으로 추가해주는게 더 깔끔할것 같습니다!!

Reply:
주석 추가해서 PR날리겠습니다~